src/components/ErrorBoundary/Error.jsx

```diff
@@ -1,38 +1,56 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { withRouter } from 'react-router'
 import styles from './Error.styl'
 import { Empty, translate } from 'cozy-ui/react'
 import brokenIcon from 'assets/icons/icon-broken.svg'
 
-const Error = props => {
-  const { t } = props
-  const update = t('Error.update')
-    .replace('#{LINK}', `<a onClick="javascript:window.location.reload(true)">`)
-    .replace('#{/LINK}', '</a>')
-  const lang = 'fr'
-  const url = `https://cozy.io/${lang === 'fr' ? 'fr' : 'en'}/support/`
-  const contact = t('Error.contact')
-    .replace('#{LINK}', `<a href="${url}" target="_blank">`)
-    .replace('#{/LINK}', '</a>')
-
-  return (
-    <div className={styles.Error}>
-      <Empty
-        className={styles.Empty}
-        icon={brokenIcon}
-        title={t('Error.title')}
-        text={
-          <div
-            dangerouslySetInnerHTML={{ __html: `${update}</br>${contact}` }}
-          />
-        }
-      />
-    </div>
-  )
-}
+class Error extends React.Component {
+  static propTypes = {
+    t: PropTypes.func.isRequired
+  }
+
+  rootRef = React.createRef()
+
+  componentDidMount() {
+    this.refreshLink = this.rootRef.current.querySelector('#error-refresh-link')
+    this.refreshLink.addEventListener('click', this.refresh)
+  }
```

Comment:
I am afraid this will not work for subsequent updates ? Maybe you have to removeEventListener and re-add also in the componentDidUpdate ?